### DISCUSSION TITLE
혹시 PR리뷰 할때 승인을 어떻게 하는지 아시나요?

### DISCUSSION THREAD

**Discussion by chadseok** [wanted-pre-onboarding-side-project/habit-tracker#37] [Q&A] [answered]
제가 만약 merge를 승인을 한다면 comment남길 때 따로 해야되는게 있을까요?

**Comment by jiyeon2** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/18395475/231392773-a64e0889-bd73-4cd7-b142-33333bbdc285.png)

별도 코멘트가 없고 바로 승인하시겠다면
우측 작은 창에서 Approve를 선택하고 submit review 하시면 될거같습니다

추가 의사결정을 기다리는 단계에서는 바로 approve 하실 필요는 없습니다!

**Reply by chadseok** [reply]
아 감사합니다!
